src/lib/notificationWorker.js

```diff
@@ -7,19 +7,18 @@ const tweetnacl = require('tweetnacl')
 const makeCaseAttestation = require('five-bells-shared/utils/makeCaseAttestation')
 const UriManager = require('./uri')
 const Log = require('./log')
-const Config = require('./config')
+const config = require('./config')
```

Comment:
@clark800 Here I changed it to lowercase, because it's no longer representing the module, but the `config` object